roles/smartgateway/tasks/main.yml

```diff
@@ -23,6 +23,27 @@
     bridge_container_image_path: "{{ lookup('env', 'RELATED_IMAGE_BRIDGE_SMARTGATEWAY_IMAGE') | default('quay.io/infrawatch/sg-bridge:latest', true) }}"
   when: bridge_container_image_path is undefined
 
+- name: Check for existing cookie secret
```

Comment:
The cookie secret is not really used when only configured for token auth, but is required for oauth-proxy to start up. kube-rbac-would not require this, if we could sort out the image handling for it.